roles/idrac_os_deployment/README.md

```diff
@@ -423,7 +423,7 @@ Role Variables
     </tr>
     <tr>
       <td>idrac_os_deployment_supported_os</td>
-      <td>{ RHEL: ["8", "9"], ESXI: ["8"] }</td>
+      <td>{ RHEL: ["8", "9"], ESXI: ["8"], UBUNTU: ["jammy"] }</td>
```

Comment:
i wonder naming `22.04` vs `jammy`

Reply:
Since in the versioning format, he has not considered patch, both seems to be ok. Still if we have chanceof having different kickstsrt template based on versions, i would recommend to go with `22.04`

Reply:
For all the Ubuntu jammy versions, the kickstarter(auto-install) template is the same